src/lib/ui/utils.ts

```diff
@@ -25,3 +28,6 @@ export const readClipboard = async () => {
 
   throw new Error('Clipboard read permission denied');
 };
+
+export const toPercentage = (value: BigNumber.Value, decimalPlaces = 2) =>
+  `${toLocalFormat(new BigNumber(value).times(100), { decimalPlaces })}%`;
```

Comment:
```suggestion
export const toPercentage = (value?: BigNumber.Value, fallback = '-', decimalPlaces = 2) =>
  value ? `${toLocalFormat(new BigNumber(value).times(100), { decimalPlaces })}%` : fallback;
```

and you can get rid of isDefined conditions where you use it